src/VisualStudio/Core/Impl/CodeModel/InternalElements/AbstractCodeMember.cs

```diff
@@ -152,7 +152,7 @@ public EnvDTE80.vsCMOverrideKind OverrideKind
             }
         }
 
-        internal virtual ImmutableArray<IParameterSymbol> GetParameters()
+        internal virtual ImmutableArray<SyntaxNode> GetParameters()
```

Comment:
Are there any overrides that don't just do `return ImmutableArray.CreateRange(CodeModelService.GetParameterNodes(LookupNode()));`?  Perhaps drop the virtual and remove the overrides?